docs/sources/reference/commandline/cli.md

```diff
@@ -1100,6 +1100,10 @@ and Docker images will report:
 
     untag, delete
 
+The `--since` and `--until` parameters can be Unix timestamps, RFC3339
+dates or Go duration strings (e.g. `10m`, `1h30m`) computed relative to
+client machine’s time.
```

Comment:
*the* client machine's time?

Perhaps we should explain "how" they are calculated, i.e., that the duration is *subtracted* from the current time?